tests/integration/test_reddit.py

```diff
@@ -144,3 +150,24 @@ def test_top(self):
         with self.recorder.use_cassette('TestDomainListing.test_top'):
             submissions = list(self.reddit.domain('youtube.com').top())
         assert len(submissions) == 100
+
+
+class TestRedditRequestor(IntegrationTest):
```

Comment:
It looks like this could be a unit test as it doesn't actually result in issuing a request.